DEVELOP.md

```diff
@@ -1,3 +1,19 @@
+# Run tests
+
+Run CrateDB on port 44200.
+```shell
+docker run --rm -it --publish=44200:4200 crate:5.1.1 \
+  -Cdiscovery.type=single-node \
+  -Ccluster.routing.allocation.disk.threshold_enabled=false
+```
+
+Install project and invoke test suite.
+```shell
+bundle install
+bundle exec rspec
```

Comment:
I was trying to run the tests locally on this branch, but it didn't work:
`/Users/user/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/activesupport-4.1.16/lib/active_support/core_ext/object/duplicable.rb:82:in <class:BigDecimal>: undefined method new for BigDecimal:Class (NoMethodError)`

Will this be fixed with #16?

Reply:
Yes, I've made it pretty far. I think making it work completely, and successfully, is almost around the corner. If you like, and if you can spare some minutes, I will be happy about any help on #16.